### PR TITLE
Handle some of breaking none cases

### PR DESCRIPTION
**Pull request by olejandro**
This PR handles some case where either a `None` value or an empty input table would cause the code to break.

### CODE REVIEW

File: times_reader/transforms.py [hunk 1]
```diff
@@ -351,6 +351,55 @@ def has_formulas(table):
...
+    """
+    Return a modified copy of 'tables' where empty tables have been deleted from the list.
+
+    :param tables:      List of tables in EmbeddedXlTable format.
+    :return:            List of non-empty tables in EmbeddedXlTable format.
+    """
+
+    check_list = [
+        datatypes.Tag.comagg,
+        datatypes.Tag.comemi,
```
Comment (siddharth-krishna) [inline times_reader/transforms.py:367]: Just curious: why not all tags? Also perhaps this is the kind of info that could also go into the meta/config file for table tags?

File: times_reader/transforms.py [hunk 2]
```diff
@@ -924,82 +974,28 @@ def remove_invalid_values(
...
-            uc_sets={},
-            sheetname="",
-            range="",
-            filename="",
-            dataframe=DataFrame({"units": sorted(process_cap_units)}),
-        )
-    )
+    for invalid_value in invalid_values:
+        if invalid_value in all_units:
+            all_units.remove(invalid_value)
```
Comment (siddharth-krishna) [inline times_reader/transforms.py:996]:
```suggestion
    all_units -= set(invalid_value)
```

Or even:
```
    all_units -= {"", "None"}  # Remove invalid values
```

File: times_reader/transforms.py [hunk 3]
```diff
@@ -924,82 +974,28 @@ def remove_invalid_values(
...
+        datatypes.Tag.currencies: ["currency"],
+    }
 
-        if table.tag == datatypes.Tag.currencies:
-            currencies.update(table.dataframe["currency"].unique())
+    invalid_values = ["", None]
 
-    tables.append(
-        datatypes.EmbeddedXlTable(
-            tag="~UNITS_ACT",
```
Comment (siddharth-krishna) [inline times_reader/transforms.py:954]: These other table tags are not required?

**Review by siddharth-krishna — APPROVED** [review]
Thanks! Just some minor questions/suggestions

File: times_reader/transforms.py (hunk 1 above)
Comment (olejandro) [reply times_reader/transforms.py:367]: Yes, I think we should handle it through meta data, i.e. defining what constitutes a non-empty table. Some of the tables define single values and don't have column names, so they should be treated differently.

File: times_reader/transforms.py (hunk 2 above)
Comment (olejandro) [reply times_reader/transforms.py:996]: Thanks, this looks neat!

File: times_reader/transforms.py (hunk 3 above)
Comment (olejandro) [reply times_reader/transforms.py:954]: No, the outputs that rely on them rely on more general tables now.

File: times_reader/transforms.py (hunk 2 above)
Comment (olejandro) [reply times_reader/transforms.py:996]: @siddharth-krishna could you please add `s` to `invalid_value` in your suggestion above? Can't find how to modify it...

**Comment by olejandro** [comment]
@siddharth-krishna I had to modify the benchmarks to make this one pass, but I can see that some of the GAMS runs are failing. I'll merge after fixing them.

**Pull request merged**